index.html

```diff
@@ -1,9 +1,9 @@
-<!doctype html>
+<!Doctype html>
```

Comment:
This actually makes me think that having a standard for HTML would be a good idea in a group. Although I personally don't mind ALLCAPS or alllowercase, just capitalizing the first letter would make me think I'm doing something wrong.

Reply:
Looking back over it, I'm not sure I'm confident with merging this; for a branch that was intending to add screen readers, it also made substantive changes to the main page. It removed content that (from my understanding) was unrelated to the issue at hand.

Reply:
@profounddark the capitalization threw me off a bit. For a second I thought maybe I had edited that as well.

Reply:
@profounddark I have gone ahead and capitalized.